### PR TITLE
QA-436 / QA-437 / QA-438 Move, Copy and Remove sentences implementations

### PR DESCRIPTION
**Pull request by miguelcleverti**
Implementations of the Move/Copy/Remove sentences for PlatfomUI BDD.

Jira Issues:
https://jira.ez.no/browse/QA-436
https://jira.ez.no/browse/QA-437
https://jira.ez.no/browse/QA-438

depends on:
ezsystems/behatbundle#28

### CODE REVIEW

**Comment by miguelcleverti** [comment]
@andrerom @pcardiga @micszo

**Comment by miguelcleverti** [comment]
:triumph: ezrobot :triumph:

**Comment by andrerom** [comment]
+1


Side, there is no exception on the loading spin function after x time, it probably should, ref
```
         while ($page->find('css', $loadingSelector) != null) {
             usleep(100 * 1000); // 100ms
         }
```

File: Features/Context/SubContext/Authentication.php [hunk 1]
```diff
@@ -51,6 +51,7 @@ public function goToPlatformUiAndLogIn($username, $password)
 
     /**
      * @Given I am logged in as admin on PlatformUI
+     * @Given I am logged in as an Editor in PlatformUI
      */
     public function loggedAsAdminPlatformUi()
```
Comment (joaoinacio) [inline Features/Context/SubContext/Authentication.php]: Nitpick / OT: 
Not necessarily admin user, maybe `loggedInAsUserInPlatformUi()` ?
Also IMO since the trait is using vars from the context instance, at least document it?

**Comment by miguelcleverti** [comment]
@andrerom This is a known issue, the waiting mechanism will mostly need to be refactored and it is not really related to this PR in specific

**Comment by andrerom** [comment]
Miguel: ok, ping when Joao is fine with it as well so we have 2x +1, and I'll merge (but the other one first)

File: Features/Context/SubContext/Authentication.php (hunk 1 above)
Comment (miguelcleverti) [inline Features/Context/SubContext/Authentication.php]: These involves changes in various places, a dedicated PR would be better

Comment (andrerom) [inline Features/Context/SubContext/Authentication.php]: I agree, could one of you do a separate PR on this once we have travis up and running for PR sanity?

**Comment by andrerom** [comment]
Ok, thanks everyone :)
